cmd/d8s/copy.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"net/http"
+
+	"github.com/google/subcommands"
+
+	"github.com/google/go-containerregistry/authn"
+	"github.com/google/go-containerregistry/name"
+	"github.com/google/go-containerregistry/v1/remote"
+)
+
+type copyCmd struct{}
+
+func (*copyCmd) Name() string { return "copy" }
+func (*copyCmd) Synopsis() string {
+	return "Lazily copies a remote src ref to a remote dst ref"
```

Comment:
nit: "Lazily" is an implementation detail, how about "Efficiently"?

Reply:
Done